boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp_iron.dts

```diff
@@ -8,12 +8,24 @@
 
 /delete-node/&cpurad_rx_partitions;
 /delete-node/&cpuapp_rx_partitions;
-/delete-node/&cpusec_cpuapp_ipc;
 
 /* This is not yet an exhaustive memory map, and contain only a minimum required to boot
  * the application core.
  */
 
+/* Temporary until NCSDK-32158 */
+&cpusec_cpuapp_ipc {
+	status = "okay";
+	mbox-names = "tx", "rx";
+	tx-region = <&cpuapp_cpusec_ipc_shm>;
+	rx-region = <&cpusec_cpuapp_ipc_shm>;
+};
+
+/* Temporary until NCSDK-32158 */
+&cpusec_bellboard {
+	status = "okay";
```

Comment:
Don't we want to enable these in individual samples, as before?

Reply:
fixed